docker-compose.yml.example

```diff
@@ -0,0 +1,25 @@
+version: '2.1'
+services:
+  app:
+    build:
+      context: .
+      args:
+        BUILD_ENV: development
+
+    restart: unless-stopped
+
+    volumes:
+      - .:/opt/fab
```

Comment:
I'm running into `Permission denied @ dir_s_mkdir - /opt/fab/tmp/cache/assets/sprockets/v3.0/bf` with these volumes. I believe the permissions of tmp on my host are overwriting the permissions of tmp is the container.

I can avoid this error by adding `- /opt/fab/tmp` to the list of volumes for the reason described here: https://stackoverflow.com/questions/30043872/docker-compose-node-modules-not-present-in-a-volume-after-npm-install-succeeds

Reply:
done